components/org.wso2.carbon.status.dashboard.core/src/main/java/org/wso2/carbon/status/dashboard/core/dbhandler/StatusDashboardWorkerDBHandler.java

```diff
@@ -79,12 +82,73 @@ public StatusDashboardWorkerDBHandler() {
                     "dashboard.");
         }
     }
 
+    private void creteConfigurationDB() {
+        Map<String, String> attributesTypeMap = workerAttributeTypeMap.get(WORKER_CONFIG_TABLE);
+        String condition =
+                "WORKERID " + attributesTypeMap.get("WORKERID") + " PRIMARY KEY," +
+                        "HOST " + attributesTypeMap.get("HOST") + "," +
+                        "PORT " + attributesTypeMap.get("PORT");
+        String resolvedTableCreateQuery = tableCreateQuery.replace(PLACEHOLDER_TABLE_NAME, WORKER_CONFIG_TABLE).replace
+                (PLACEHOLDER_COLUMN, condition);
+        Connection conn = this.getConnection();
+        try {
+            PreparedStatement stmt = conn.prepareStatement(resolvedTableCreateQuery);
+            stmt.execute();
+            stmt.close();
+        } catch (SQLException e) {
+            throw new RDBMSTableException("Error creating table." + WORKER_CONFIG_TABLE);
+        }
+    }
+
+    private void creteDetailsDB() {
+        Map<String, String> attributesTypeMap = workerAttributeTypeMap.get(WORKER_DETAILS_TABLE);
+        String condition =
+                        "CARBONID "+attributesTypeMap.get("CARBONID") +" PRIMARY KEY , " +
+                        "WORKERID "+attributesTypeMap.get("WORKERID") +", " +
+                        "JAVARUNTIMENAME "+attributesTypeMap.get("JAVARUNTIMENAME") +", " +
+                        "JAVAVMVERSION "+attributesTypeMap.get("JAVAVMVERSION") +", " +
+                        "JAVAVMVENDOR "+attributesTypeMap.get("JAVAVMVENDOR") +", " +
+                        "JAVAHOME "+attributesTypeMap.get("JAVAHOME") +", " +
+                        "JAVAVERSION "+attributesTypeMap.get("JAVAVERSION") +", " +
+                        "OSNAME "+attributesTypeMap.get("OSNAME") +", " +
+                        "OSVERSION "+attributesTypeMap.get("OSVERSION") +", " +
+                        "USERHOME "+attributesTypeMap.get("USERHOME") +", " +
+                        "USERTIMEZONE "+attributesTypeMap.get("USERTIMEZONE") +", " +
+                        "USERNAME "+attributesTypeMap.get("USERNAME") +", " +
+                        "USERCOUNTRY "+attributesTypeMap.get("USERCOUNTRY") +", " +
+                        "REPOLOCATION "+attributesTypeMap.get("REPOLOCATION") +", " +
+                        "SERVERSTARTTIME "+attributesTypeMap.get("SERVERSTARTTIME") +", " +
+                        "LASTSNAPSHOTTIME "+attributesTypeMap.get("LASTSNAPSHOTTIME") +", " +
+                        "FOREIGN KEY (WORKERID) REFERENCES WORKERS_CONFIGURATION(WORKERID)";
+
+        String resolvedTableCreateQuery = tableCreateQuery.replace(PLACEHOLDER_TABLE_NAME, WORKER_DETAILS_TABLE).replace
+                (PLACEHOLDER_COLUMN, condition);
+        Connection conn = this.getConnection();
+        try {
+            PreparedStatement stmt = conn.prepareStatement(resolvedTableCreateQuery);
+            stmt.execute();
+            stmt.close();
```

Comment:
Close `stmt` in the `finally` block

Reply:
Fixed cc32da4fdfe699567c47b82df7382c8f8618b857